server/crashmanager/management/commands/remove_tool_from_user.py

```diff
@@ -0,0 +1,42 @@
+from django.contrib.auth.models import User as DjangoUser
```

Comment:
The command should be called `rm_tool_from_user` to be more like other command names.